File: frontend/src/supabaseClient.js
import { createClient } from '@supabase/supabase-js';

const supabaseUrl = 'https://vmvqdzxvbmhugolcyojw.supabase.co';
const supabaseKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZtdnFkenh2Ym1odWdvbGN5b2p3Iiwicm9sZSI6ImFub24iLCJpYXQiOjE3NDg5MDkxMzMsImV4cCI6MjA2NDQ4NTEzM30.CdMzMmHSGmshPAse2NzK7mvqHcegpe3-HfTc6wdNWq8';

export const supabase = createClient(supabaseUrl, supabaseKey);

File: backend/src/core/supabase_client.py
# Configuração do cliente Supabase
import os
from supabase import create_client
from dotenv import load_dotenv

# Carrega variáveis de ambiente
load_dotenv()

# Obtém credenciais do Supabase das variáveis de ambiente
SUPABASE_URL = os.getenv("SUPABASE_URL")
SUPABASE_KEY = os.getenv("SUPABASE_KEY")

# Verifica se as credenciais estão definidas
if not SUPABASE_URL or not SUPABASE_KEY:
    raise ValueError("Credenciais do Supabase não encontradas nas variáveis de ambiente")

# Cria e exporta o cliente Supabase
supabase = create_client(SUPABASE_URL, SUPABASE_KEY)
